packages/components/header/header.js

```diff
@@ -35,11 +36,10 @@ class Header {
 
     this.setupMobileMenu()
     this.setupNavigation()
-    this.calculateBreakpoints()
     this.updateNavigation()
 
     this.handleResize = this.debounce(() => {
-      this.calculateBreakpoints()
+      this.resetNavigation()
```

Comment:
minor thing so feel free to ignore, but the naming here confused me for a bit as resetNavigation and updateNavigation sound similar, and also `resetNavigation` has two side effects now (updating `this.navigationItems`, and moving the elements in the DOM)

I guess an alternate approach would be to leave this as `calculateBreakpoints`, remove the short circuiting behaviour, then have `updateNavigation` handle all the DOM manipulation.

I'm thinking in the "Move items based on available width" loop you would first moving everything back to the navigationList, then move the overflowing items to the mobile menu.